static/panes/haskellstg-view.ts

```diff
@@ -67,7 +67,7 @@ export class HaskellStg extends MonacoPane<monaco.editor.IStandaloneCodeEditor,
     }
 
     override getDefaultPaneName(): string {
-        return 'Haskell STG viewer';
+        return 'GHC STG viewer';
```

Comment:
Also this V. Sorry about that, we're trying to nromalize all the panes into the same style :)